CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ if( T8CODE_ENABLE_MPI )
     if( NOT MPIEXEC_EXECUTABLE )
         message( FATAL_ERROR "MPIEXEC was not found" )
     endif()
-    set( SC_ENABLE_MPI ON )
```

Comment:
Could you explain why you comment out this line?

Reply:
Currently it is not the `SC_ENABLE_MPI` flag which enables mpi support in libsc but the unscoped `mpi` flag. I am currently working on fixing this in libsc (among other things) and this should be merged in the develop quickly